kayenta-web/src/test/java/com/netflix/kayenta/controllers/CanaryConfigControllerTest.java

```diff
@@ -63,9 +63,30 @@ public void getCanaryConfig_returnsBadRequestResponseForNotResolvedAccount() thr
                 "/canaryConfig/{configId}?configurationAccountName={account}",
                 CONFIG_ID,
                 "unknown-account"))
-        .andDo(print())
         .andExpect(status().isBadRequest())
         .andExpect(content().contentType("application/json"))
         .andExpect(jsonPath("$.message", equalTo("Unable to resolve account unknown-account.")));
   }
+
+  @Test
+  public void storeCanaryConfig_returnsBadRequestResponseForMissingRequestBody() throws Exception {
+    this.mockMvc
+        .perform(
+            post("/canaryConfig?configurationAccountName={account}", CONFIGS_ACCOUNT)
+                .contentType(APPLICATION_JSON))
+        .andExpect(status().isInternalServerError()); // expected: bad request ( should be fixed by:
+    // https://github.com/spinnaker/kork/pull/535)
```

Comment:
This test is still failing even though that is merged, I think orca will need to be bumped so that Kork is updated

- [ ] tests pass